Source/SwiftLintFramework/Rules/MissingDocsRule.swift

```diff
@@ -54,17 +54,9 @@ extension File {
 }
 
 public enum AccessControlLevel: String {
-    case Private = "private"
```

Comment:
I could't actually find specific instances where `description` was used on `AccessControlLevel`, but that's easy to miss since there are several way `description` can be used implicitly that's hard to search for. I went ahead and made it conform to `CustomStringConvertible` as suggested which returns the short strings. I also had to modify `MissingDocsRule.init(configuration:)` because that relied on the rawValue being the shorter version of the string:

```Swift
let acl = array.flatMap(AccessControlLevel.init(description:))
```

Since `AccessControlLevel` needs to be able to be instantiated from the short strings I also head to create an init method for that. Couldn't think of a better name so hopefully `init(description:)` will suffice.